### ISSUE TITLE
cannot find symbol             result.updateName(ITEM_OBF, new StringHolder(getString(R.string.open_food_drawer)));

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#2865]
```  
Compiling with JDK Java compiler API.
/home/travis/build/openfoodfacts/openfoodfacts-androidapp/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java:398: error: cannot find symbol
            result.updateName(ITEM_OBF, new StringHolder(getString(R.string.open_food_drawer)));
                                                                           ^
  symbol:   variable open_food_drawer
  location: class string
```

**Comment by VaiTon** [comment]
Could you give us the environment to reproduce? :-)

**Comment by deniger** [comment]
@teolemon Fixed with #2869. Correct ?

**Comment by teolemon** [comment]
yup, thanks :-)
